traversal/planner/GraphPlanner.java

```diff
@@ -385,7 +400,7 @@ public String toString() {
         return str.toString();
     }
 
-    private void initialiseOptimiserValues() {
+    private void initialiseVertexOrderGreedy() {
```

Comment:
This method no longer initialises all the opimiser variables. Vertex ordering variables are the main ones in the solution, all the rest are auxiliary, inferred from the ordering for the purposes of calculating the objective function value. This inference is performed elsewhere, viz., in `setOptimiserValues()` in `updateOptimiser()`.